src/pages/en/guides/deploy.md

```diff
@@ -474,6 +474,44 @@ $ npm run build
 $ 0 deploy
 ```
 
+## Deno
+
+Deploy an Astro site on Deno using the Deno adapter & [deployctl](https://github.com/denoland/deployctl)
+
+1. Install Deno Adapter in Astro
+
+Install the Deno (beta) adapter with `npm i -D @astrojs/deno`.
+
+Change your astro.config.mjs configuration file to the following:
+
+```js
+import { defineConfig } from 'astro/config';
+import deno from "@astrojs/deno";
+
+export default defineConfig({
+  adapter: deno()
+});
+```
+
+2. Preview your local build with Deno
```

Comment:
Do we need another step first to actually run the build command? Might seem obvious but if you’re following step-by-step, you could forget.

````suggestion
2. Build your site by running `astro build`

3. Preview your local build with Deno
````